pom.xml

```diff
@@ -79,6 +79,19 @@
 		</dependency>
 
 
+		<dependency>
+			<groupId>org.postgresql</groupId>
+			<artifactId>postgresql</artifactId>
+			<version>42.3.8</version>
```

Comment:
Are these versions the latest? It is ok if not, but need to watch if these versions are out of maintenance or don't work with the latest postgresql/my db.

Reply:
Those versions are latest both mysql and postgresql